File: Examples/DotNET/CSharp/Files/CheckFilePresence.cs
using System;
using Com.Aspose.Storage.Api;
using Com.Aspose.Storage.Model;

namespace Files
{
    class CheckFilePresence
    {
        public static void Run()
        {
            // ExStart:1           
            StorageApi storageApi = new StorageApi(Common.APP_KEY, Common.APP_SID, Common.BASEPATH);

            string Path = "testfile.txt";
            string versionId = null;
            string storage = null;
           
            try
            {
                // Invoke Aspose.Storage Cloud SDK API to check file presence                
                Com.Aspose.Storage.Model.FileExistResponse apiResponse = storageApi.GetIsExist(Path, versionId, storage);
                if (apiResponse != null && apiResponse.Status.Equals("OK"))
                {
                    Console.WriteLine(apiResponse.FileExist.ToString());
                    Console.ReadKey();
                }
            }
            catch (Exception ex)
            {
                System.Diagnostics.Debug.WriteLine("error:" + ex.Message + "\n" + ex.StackTrace);
            }
            // ExEnd:1
        }
    }
}

File: Examples/PHP/Files/CheckFilePresence.php
//ExStart:
<?php
require_once realpath(__DIR__ . '/..') . '/vendor/autoload.php';
require_once realpath(__DIR__ . '/..') . '/Utils.php';

use Aspose\Storage\StorageApi;
use Aspose\Storage\AsposeApp;

class Files {

	public $storageApi;

	public function __construct() {
		AsposeApp::$appSID = Utils::appSID;
		AsposeApp::$apiKey = Utils::apiKey;
		$this->storageApi = new StorageApi();
	}

	public function checkFilePresence() {
		// Upload file to Aspose Cloud Storage
		$fileName = "testfile.txt";
		Utils::uploadFile($fileName);

		$result = $this->storageApi->GetIsExist($Path=$fileName, $versionId = null, $storage = null);
        print_r($result);
	}
}

$files = new Files();
$files->checkFilePresence();

?>
//ExEnd: